documentation/asciidoc/accessories/ai-camera/details.adoc

```diff
@@ -245,18 +245,18 @@ There are a number of scaling/cropping/translation operations occurring from the
 | Returns the input tensor size based on the neural network model used.
 
 | `IMX500.get_outputs(metadata)`
-| Returns the output tensors from the Picamera2 image metadata metadata.
+| Returns the output tensors from the Picamera2 image metadata.
 
 | `IMX500.get_output_shapes(metadata)`
 | Returns the shape of the output tensors from the Picamera2 image metadata for the neural network model used.
 
 | `IMX500.set_inference_roi_abs(rectangle)`
-| Sets the region of interest (ROI) crop rectangle which determines which part of the sensor image is converted to the input tensor that is used for inferencing on the IMX500. The region of interest should be specified in units of pixels at the full sensor resolution, as a `(x_offset, y_offset, width, height)` tuple.
+| Sets the ROI crop rectangle which determines which part of the sensor image is converted to the input tensor that is used for inferencing on IMX500. The ROI should be specified in units of pixels at the full sensor resolution, as a `(x_offset, y_offset, width, height)` tuple.
 
 | `IMX500.set_inference_aspect_ratio(aspect_ratio)`
-| Automatically calculates region of interest (ROI) crop rectangle on the sensor image to preserve the given aspect ratio. To make the ROI aspect ratio exactly match the input tensor for this network, use `imx500.set_inference_aspect_ratio(imx500.get_input_size())`.
+| Automatically calculates ROI crop rectangle on the sensor image to preserve the given aspect ratio. To make the ROI aspect ratio exactly match the input tensor for this network, use `imx500.set_inference_aspect_ratio(imx500.get_input_size())`.
```

Comment:
We prefer to expand definitions in our reference material since some users might skip the prose altogether and require the expansion.